### PR TITLE
Maintenance: Update packages

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
_An update a day keeps the hackers away!_

This is just some basic maintenance to update `ora`, `ajv`, and `better-ajv-errors` to their latest versions.  This isn't really meant to add any additional functionality from their new versions, but I figured we might as well update them to the latest releases.

`ajv` changelog (since 6.6.2):

- Option `strictDefaults` to report ignored defaults
- Option `strictKeywords` to report unknown keywords
- OpenAPI keyword `nullable` can be any boolean (and not only `true`).
- Custom keyword definition changes:
  - `dependencies` option in to require the presence of keywords in the same schema.
  - more strict validation of the definition using JSON Schema.
- Docs: (security considerations)[https://github.com/epoberezkin/ajv#security-considerations].
- [Meta-schema](https://github.com/epoberezkin/ajv/blob/master/lib/refs/json-schema-secure.json) for the security assessment of JSON Schemas.
- Option `useDefaults: "empty"` to replace `null` and `""` (empty strings) with default values (in addition to assigning defaults to missing and undefined properties).
- Update draft-04 meta-schema to remove incorrect usage of "uri" format.

`ora` changelog (since 3.0.0):

- Support changing the spinner while `ora` is running
- Add `indent` option
- Add TypeScript definition
- Add `prefixText` option
- Fix regression with `stopAndPersist` output
- Refactor TypeScript definition to CommonJS compatible export

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,11 +29,11 @@
   },
   "homepage": "https://github.com/mdn/browser-compat-data#readme",
   "devDependencies": {
-    "ajv": "^6.6.2",
+    "ajv": "^6.10.0",
     "better-ajv-errors": "^0.6.1",
```
Comment (ExE-Boss) [inline package.json]:
```suggestion
     "better-ajv-errors": "^0.6.2",
```

Comment (vinyldarkscratch) [reply package.json]: Whoops, I must have missed this one!

**Comment by ExE-Boss** [comment]
You’ll need to update `package-lock.json` with:
```diff
diff --git a/package-lock.json b/package-lock.json
index 7b494dbe5a..00826aad15 100644
--- a/package-lock.json
+++ b/package-lock.json
@@ -262,9 +262,9 @@
       }
     },
     "better-ajv-errors": {
-      "version": "0.6.1",
-      "resolved": "https://registry.npmjs.org/better-ajv-errors/-/better-ajv-errors-0.6.1.tgz",
-      "integrity": "sha512-UaaWknKQQRSPT/zF+pYKdS7swLf/0NbPll52/bfYlpM72+iFFC49cBErGRxAWHC9AOsD0yy176DR/JyaIRD8+Q==",
+      "version": "0.6.4",
+      "resolved": "https://registry.npmjs.org/better-ajv-errors/-/better-ajv-errors-0.6.4.tgz",
+      "integrity": "sha512-+spBhtcCzovXWeHpt5dGylFsn3p5l9w+KcUqh/b4MFdLV+q1sT1olxD9izvwi0D3WuP06eVgeZAGLtxtTnUIDg==",
       "dev": true,
       "requires": {
         "@babel/code-frame": "^7.0.0",
```

**Comment by vinyldarkscratch** [comment]
I’ll run that update to package-lock shortly.  😉

File: package.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
   "homepage": "https://github.com/mdn/browser-compat-data#readme",
   "devDependencies": {
     "ajv": "^6.10.0",
-    "better-ajv-errors": "^0.6.1",
+     "better-ajv-errors": "^0.6.4",
```
Comment (ExE-Boss) [inline package.json]: For whatever reason, GitHub decided to insert a space here, despite the suggestion displaying it without said space.

**Pull request merged**